unidock/README.md

```diff
@@ -19,26 +19,26 @@ Developed by [DP Technology](https://dp.tech/en), [Hermite®](https://dp.tech/en
 
 Uni-Dock is now available on the new-generation drug computing design platform [Hermite®](https://dp.tech/en/product/hermite) for ultralarge virtual screening.
 
-For further cooperations on developing Uni-Dock and trying out Hermite®, please contact us at bd@dp.tech .
+For further cooperations on developing Uni-Dock and trying out Hermite®, please contact us at <bd@dp.tech> .
 
 ## Installation
 
-Uni-Dock supports NVIDIA GPUs on Linux platform.
-[CUDA toolkit](https://developer.nvidia.com/cuda-downloads) is required.
+Uni-Dock officially supports NVIDIA GPUs with [compute capability](https://en.wikipedia.org/wiki/CUDA#GPUs_supported) >= 7.0 on Linux platform.
```

Comment:
> Compute Capability of RTX 20 and T4 are 7.5, so should we support them officially? (in case they are not supported at [unidock/CMakeLists.txt](https://github.com/dptech-corp/Uni-Dock-Dev/pull/20/files#diff-564d61b6520a4ac5011f1bc2e6b4ddee648dc30a19bba142c930df7c37829225).)

@zhengh96 Yes. Not presenting in CMakeLists.txt does not mean we don't support the architecture, and the executable built is still available.